src/util/irep.cpp

```diff
@@ -923,6 +923,49 @@ std::size_t irept::full_hash() const
 
 /*******************************************************************\
 
+Function: irept::stable_hash
+
+  Inputs:
+
+ Outputs:
+
+ Purpose: Stable hash, hashes the actual content of the string. 
```

Comment:
This is a bit confusing: it will hash the actual contents of all contained strings (as is, the comment would rather fit `stable_hash_string` than `irept::stable_hash`).